src/main/java/codeu/model/data/Conversation.java

```diff
@@ -61,4 +63,16 @@ public String getTitle() {
   public Instant getCreationTime() {
     return creation;
   }
+  /** Time display */ 
+  public String getTime() {
+    LocalDateTime localDate = LocalDateTime.ofInstant(creation, ZoneId.systemDefault());
+    int hr = localDate.getHour();
+    Boolean AM=true;
+    if (hr > 12) {
+      hr = hr % 12;
+      AM = false;
+    }
+    String Date = localDate.getMonth().toString() + " " + localDate.getDayOfMonth() + ", " + localDate.getYear() + " ~ " + hr + ":" + localDate.getMinute() + " " + (AM ? "AM":"PM");
+    return Date;
```

Comment:
Nit: you don't need to store this variable at all! Could just do:
`return localDate.getMonth().toString() + " " + localDate.getDayOfMonth() ...`